react-carousel-component/my-app/src/RotatingBanner.js

```diff
@@ -0,0 +1,51 @@
+import NavigationButton from './NavigationButton';
+import IndexButtons from './IndexButtons';
+import Banner from './Banner';
+import './RotatingBanner.css';
+import { useState, useEffect } from 'react';
+
+export default function RotatingBanner({ items }) {
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  useEffect(() => {
+    const id = setInterval(handleIncrement, 3000);
+    console.log('effect');
+
+    return () => clearInterval(id);
+  }, [handleIncrement]);
```

Comment:
Don't you get a lint warning that `handleIncrement` will cause the effect to execute on every render? If it doesn't, you may not have your environment set up correctly to give you React lint warnings.

Do you remember how to fix that?

Reply:
Yes, it seems like the lint is showing now